spec/smart_notifications.adoc

```diff
@@ -0,0 +1,45 @@
+[#smart_notifications]
+= Smart notifications
+
+== Overview
+
+Smart notifications, sometimes called dev notifications, allow developers using xref:glossary.adoc#connected_mode[Connected Mode] in SonarLint to receive in-IDE notifications from SonarQube or SonarCloud when:
+
+* the Quality Gate status (failed/success) of a project open in the IDE changes
+* a SonarQube or SonarCloud analysis raises new xref:glossary.adoc#finding[findings] introduced by this developer in a project open in the IDE
+
+SonarLint polls the server (SQ/SC) every minute to retrieve any new notifications up to 1 day old. When the polling has been done, the timestamp of the polling date is stored and will be used next time as a reference, so the client cannot retrieve the same notification later.
```

Comment:
I know that @henryju does not like the term "server". We probably need to find a term for this concept

Reply:
maybe simply "SonarLint will poll SonarQube or SonarCloud"